source/docs/v3/tenant-attribute-encryption.blade.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Tenant attribute encryption
+extends: _layouts.documentation
+section: content
+---
+
+# Tenant attribute encryption {#encrypt}
+
+To encrypt attributes on the Tenant model, store them in [custom columns](https://tenancyforlaravel.com/docs/v3/tenants/#custom-columns) and cast the attributes to `'encrypted'`, or your custom encryption cast.
+
+For example, we'll encrypt the tenant's database credentials – `tenancy_db_username` and `tenancy_db_password`. We need to create custom columns for these attributes, because they are, by default, stored using [virtual column](https://github.com/archtechx/virtualcolumn).
```

Comment:
`stored using virtual column` is kind of clunky.
```suggestion
For example, we'll encrypt the tenant's database credentials – `tenancy_db_username` and `tenancy_db_password`. We need to create custom columns for these attributes, because by default, they are stored in the [virtual `data` column](https://github.com/archtechx/virtualcolumn).
```

Also I'm not sure about linking to that package. Do we really not explain the virtual data column in our docs? It'd be more appropriate to link to a page in Tenancy docs than linking to the package.

Reply:
> It'd be more appropriate to link to a page in Tenancy docs than linking to the package

I can't find anything that directly mentions the package in the Tenancy docs. Only the `data` column (https://tenancyforlaravel.com/docs/v3/tenants/#custom-columns), but making "virtual `data` column" link to that isn't ideal because "custom columns" is already linked to the same URL. So I think changing "virtual `data` column" so it doesn't link to anything might be the best.